CorsixTH/Lua/ui.lua

```diff
@@ -1066,6 +1066,15 @@ function UI:removeWindow(closing_window)
   end
 end
 
+--! Function to check if we have any must pause windows open
+--!return (bool) Returns true if a must pause window is found
+function UI:checkForMustPauseWindows()
+  for i in pairs(self.windows) do
+    if self.windows[i].mustPause() then return true end
+  end
```

Comment:
Probably better to do

```lua
for _, window in pairs(self.windows) do
  if window:mustPause() then return true end
```
note the colon not the full stop.

Same issues manifest here from the previous change
https://github.com/CorsixTH/CorsixTH/blob/b7e6806f6953dee7bce47e5ce39eee8e4436faba/CorsixTH/Lua/ui.lua#L1050-L1056

But can refactor for code reuse.

Reply:
Note still
```lua
if not pauseGame and closing_window.mustPause() then
```

full stop vs colon